src/shared/infra/database/typeorm/migrations/1614849166260-CreateBusiness.ts

```diff
@@ -0,0 +1,94 @@
+import {
+  MigrationInterface,
+  QueryRunner,
+  Table,
+  TableForeignKey,
+} from 'typeorm';
+
+export class CreateBusiness1614849166260 implements MigrationInterface {
+  public async up(queryRunner: QueryRunner): Promise<void> {
+    await queryRunner.createTable(
+      new Table({
+        name: 'tb_business',
+        columns: [
+          {
+            name: 'id',
+            type: 'uuid',
+            isPrimary: true,
+            generationStrategy: 'uuid',
+            default: 'uuid_generate_v4()',
+          },
+          {
+            name: 'name',
+            type: 'varchar',
+            length: '150',
+          },
+          {
+            name: 'email_login',
+            type: 'varchar',
+            length: '45',
+          },
+          {
+            name: 'password',
+            type: 'varchar',
+            length: '25',
```

Comment:
Increase the value of the password, because the password is encrypted, it will pass the 25 houses of easy occupation